operator/builtin/parser/xml/xml_test.go

```diff
@@ -0,0 +1,146 @@
+package xml
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/observiq/stanza/entry"
+	"github.com/observiq/stanza/operator"
+	"github.com/observiq/stanza/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParse(t *testing.T) {
+	testCases := []struct {
```

Comment:
Would like to see more test cases with more complex failure cases that happens with non-closed XML tags or bad attribute fields or others.

Reply:
I'll add two more test cases for bad attributes and non closed XML tags. Let me know if you can think of any other test cases.